ci/cloudbuild/notifiers/alerts/function/function.cc

```diff
@@ -95,4 +102,14 @@ void SendBuildAlerts(google::cloud::functions::CloudEvent event) {
   auto const chat = MakeChatPayload(bs);
   std::cout << nlohmann::json{{"severity", "INFO"}, {"chat", chat}} << "\n";
   HttpPost(webhook, chat.dump());
+  std::cout << "bs = " << bs.build.dump() << std::endl;
```

Comment:
Where do these `cout`s show up?

Reply:
In the log for Cloud Run, where these are deployed.  I think I should remove this one though